From: Félix Arreola Rodríguez <fgatuno.123@gmail.com>
Description: Fix kfreebsd build
 The kfreebsd build fails due a missing defined
Forwarded: yes
Bug: http://mametesters.org/view.php?id=4127

--- a/src/osd/sdl/sdlprefix.h
+++ b/src/osd/sdl/sdlprefix.h
@@ -42,7 +42,7 @@
 /* Large file support on IRIX needs _SGI_SOURCE */
 #undef _POSIX_SOURCE
 
-#elif defined(__linux__)
+#elif defined(__linux__) || defined(__FreeBSD_kernel__)
 #define SDLMAME_LINUX 1
 
 #elif defined(__FreeBSD__)
